templates/nestjs-starter/package/index.js

```diff
@@ -0,0 +1,41 @@
+const dependencies = require('./dependencies');
+const devDependencies = require('./devDependencies');
+
+module.exports = function resolvePackage(setup, { appName, runCommand }) {
+  const packageJson = {
+    name: appName,
+    private: true,
+    version: '1.0.0',
+    description: 'Nest TypeScript starter repository',
+    license: 'MIT',
+    scripts: {
+      build: 'nest build',
+      format: 'prettier --write "**/*.{js,ts,md,json,yml,yaml}"',
+      start: 'nest start',
+      dev: 'nest start --watch',
+      'start:debug': 'nest start --debug --watch',
+      'start:prod': 'node dist/main',
```

Comment:
I think we can rename those to something like

- `start:debug` to `dev:debug`
- `start:prod` to `preview`